website/content/en/docs/User Guide/Learn Ops Management/cloud_monitoring.md

```diff
@@ -27,11 +27,24 @@ There are many pre-built monitoring pages. For example, the GKE Cluster Details
 
 You can also use the Monitoring console to create alerts and uptime checks, and to create dashboards that chart metrics you are interested in.  For example, Metrics Explorer lets you select a specific metric, configure it for charting, and then save the chart. Select **Monitoring > Metrics Explorer** from the navigation panel to bring it up.
 
+To search and view  metrics, type the name of the metric or the type of resource, for example to search [OpenCensus metrics](https://cloud-ops-sandbox.dev/docs/user-guide/#opencensus-to-become-opentelemetry) in the **Monitoring > Metrics Explorer > ** search for `grpc.io`:
+
+![image](/docs/images/user-guide/48-metrics-explorer-rpc.png)
+
 The following chart shows the client-side RPC calls that did not result in an OK status:
 
 ![image](/docs/images/user-guide/21-metrics-explorer.png)
 
-This chart uses the  metric type `custom.googleapis.com/opencensus/ grpc.io/client/completed_rpcs` (display name: "OpenCensus/grpc.io/client/ completed_rpcs" ), and filters on the  `grpc_client_status` label to keep only those time series  where the label's value is something other than "OK".
+This chart uses the  metric type `custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs` (display name: "OpenCensus/grpc.io/client/completed_rpcs" ), and filters on the  `grpc_client_status` than is equal to "OK".
+
+Below you can find another example, the below chart only display grpc_client_status that are not "OK" (e.g. PERMISSION_DENIED) and that the grpc_client_method do not include google, i.e. it's from the application services.
+
+![image](/docs/images/user-guide/49-metrics-explorer-filter-rpc.png)
+
+In addition GCP predefined [dashboards](https://cloud.google.com/monitoring/dashboards) mentioned above, Cloud Operations Sandbox come with several additional dashboards that are provisioned using [Terraform code](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring/dashboards). 
+As part of the `User Experience Dashboard` you can view a few of Opencensus metrics: `HTTP Request Count by Method`, `HTTP Response Errors` and `HTTP Request Latency, 99th Percentile` you edit the dashboard and add additional charts, and you can also open the chart in the Metrics explorer as shown below:
```

Comment:
Some rephrase suggestions: 

The following example displays results where the `grpc_client_status` is not "OK" (e.g. PERMISSION_DENIED) and where the `grpc_client_method` does not include "google", i.e. errors from application services.

Reply:
In addition to the default GCP [dashboards](https://cloud.google.com/monitoring/dashboards) mentioned above, Cloud Operations Sandbox provisions several...

Reply:
In the `User Experience Dashboard`, you can also view Opencensus metrics like `HTTP Request Count by Method`, `HTTP Response Errors` and `HTTP Request Latency, 99th Percentile`. 

Additionally, you can edit the dashboard, add additional charts, and also open the chart in the Metrics explorer as shown below: